packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureGraph/to/V1_GraphBuilderContext.ts

```diff
@@ -318,30 +319,30 @@ export class V1_GraphBuilderContext {
     pointer: V1_PropertyPointer,
   ): PropertyImplicitReference => {
     assertNonEmptyString(
-      pointer.class,
-      `Property pointer 'class' field is missing or empty`,
+      pointer.propertyOwner,
+      `Property pointer 'propertyOwner' field is missing or empty`,
     );
     assertNonEmptyString(
       pointer.property,
       `Property pointer 'property' field is missing or empty`,
     );
-    const ownerReference = this.resolveClass(pointer.class);
-    const value = getOwnClassProperty(ownerReference.value, pointer.property);
+    const ownerReference = this.resolvePropertyOwner(pointer.propertyOwner);
+    const value = getAllOwnProperty(ownerReference.value, pointer.property);
```

Comment:
this reads wrong, -> `getOwnProperty`

Reply:
There is already a function with the same name. So i named this getAllOwnProperty